custom_components/ble_monitor/const.py

```diff
@@ -319,6 +319,15 @@ class BLEMonitorBinarySensorEntityDescription(
         device_class=None,
         force_update=False,
     ),
+    BLEMonitorBinarySensorEntityDescription(
+        key="armed away",
+        sensor_class="BaseBinarySensor",
+        update_behavior="Instantly",
+        name="ble armed away",
+        unique_id="arm_",
+        device_class=BinarySensorDeviceClass.LOCK,
+        force_update=True,
+    ),
```

Comment:
See previous comment, this armed away sensor isn't used currently. It's fine to add it like you did, just curious if this is as you intended.

Reply:
Because my lock is using the arming event to pass outside the door to lock the door (handle to lift)
I was previously using this sensor to get the lock status, but have since eliminated this practice.

Reply:
Ok, but should we remove it than, to keep things clean. ?